torch/_dynamo/variables/misc.py

```diff
@@ -814,10 +919,25 @@ def call_method(
             return variables.ConstantVariable.create(None)
 
         if name != "save_for_backward":
-            unimplemented(f"autograd.Function context method: {name}")
+            unimplemented_v2(
+                gb_type="Unsupported autograd.Function context method",
+                context=f"call_method {self} {name}",
+                explanation="Dynamo does not support calling the method "
+                f"`{name}` on `autograd.Function` context objects. Supported "
+                "methods are `__setattr__`, `save_for_backward` and "
+                "`mark_non_differentiable`.",
+                hints=[*graph_break_hints.USER_ERROR],
```

Comment:
SUPPORTABLE?

Reply:
Updated in this commit: https://github.com/pytorch/pytorch/pull/152274/commits/c31e580f3bbb93613657fbff416f19d823078185